lib/logstash/inputs/gelf.rb

```diff
@@ -11,16 +11,8 @@
 # making it a good choice if you already use Graylog2 today.
 #
 # The main use case for this input is to leverage existing GELF
-# logging libraries such as the GELF log4j appender. A library used
-# by this plugin has a bug which prevents it parsing uncompressed data.
```

Comment:
Curious. Why remove this?

Also, if you hope to update the documentation, the docs for this plug now live in `docs/index.asciidoc`